buildconfig/Jenkins/Conda/build

```diff
@@ -65,15 +71,15 @@ if [[ $ENABLE_BUILD_CODE == true ]]; then
 fi
 
 if [[ $ENABLE_UNIT_TESTS == true ]]; then
-    cmake --build . --target AllTests $BUILD_OPTIONS
-fi
-
-if [[ $ENABLE_UNIT_TESTS == true || $ENABLE_SYSTEM_TESTS == true || $ENABLE_DOC_TESTS == true ]]; then
+    # this will get the test data
+    cmake --build . --target StandardTestData AllTests $BUILD_OPTIONS
```

Comment:
Am I interpreting this correctly that building the `StandardTestData` target before the `AllTests` target stops the latter downloading the test data?

Reply:
Specifying multiple targets lets the builder (e.g. ninja) figure out all the dependencies and schedule them as it sees fit. For example, specifying targets of `Kernel` and `KernelTest` is equivalent to specifying only `KernelTest` since it depends on `Kernel` in cmake. ninja figures out the order it wants to do things in.